src/components/Food-item-card/Food.jsx

```diff
@@ -1,9 +1,22 @@
-/* eslint-disable react/prop-types */
 import React from "react";
 import PropTypes from "prop-types";
 import { useState } from "react";
 import { useHistory } from "react-router-dom";
-function Food({ img, title, cal, time, id }) {
+function Food({ img, title, cal, time, id, categories,ingredients }) {
+  const storage = window.localStorage;
+  var fav = JSON.parse(storage.getItem("fav")) ? JSON.parse(storage.getItem("fav")) : []; 
```

Comment:
replace `var` with `const`
```suggestion
  const fav = JSON.parse(storage.getItem("fav")) ? JSON.parse(storage.getItem("fav")) : []; 
```